conf/evolutions/default/184.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO version VALUES ('7.13.3', now(), 'Adds a Not Sure validation filter to Gallery and LabelMap.');
+
+# --- !Downs
+DELETE FROM version WHERE version_id = '7.13.3';
```

Comment:
Why is Github saying that you added this file? It should already be there from the develop branch. Maybe try to pull in changes from develop..?